usaspending_api/api_contracts/search_filters.md

```diff
@@ -177,6 +177,30 @@ Request parameter description:
 * `date_type`:  (enum[string])
     Check specific time period objects for date_type's members, defaults, and descriptions.
 
+## Award Search Time Period Object
+
+**Description:**
+See [Time Period](#time-period)
+
+**Examples**
+See [Time Period](#time-period)
+
+Request parameter description:
++ `start_date`: (required)
+    See [Time Period](#time-period)
++ `end_date`: (required)
+    See [Time Period](#time-period)
++ `date_type`: (optional)
+    + Members
+        + `action_date`
+            This date type value is the default type compared to the `start_date`.
+        + `date_signed`
+            This date type value is the default type compared to the `end_date`.
```

Comment:
> What happens when one of these options are passed in as the date_type?

Whatever happens existed before this PR.

Reply:
> Might be good to point out that action_date corresponds to the latest transaction associated with the award, while date_signed corresponds to the earliest.

I'm not sure I understand how pointing this out relates to my PR and my story.

Reply:
> if so, is there any way to achieve the default behavior when providing a date_type?

This again seems unrelated to my PR. But doesn't "date type default" indicate that it can't be achieved unless `date_type` isn't provided?

Reply:
On all these, my feedback is around the language around these `date_type` options added in your PR. Not suggesting you change any behavior. 

This feedback relates to your PR because you added this new language to a Markdown file in your PR, and I want to make sure that the usage of this API based on this language is clear to end users.

Reply:
> On all these, my feedback is around the language around these date_type options added in your PR.

Oh ok makes sense. Let me change my mindset to thinking of these points in terms of the documentation.